workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerCreation/SendCreatePlayerRequestSystem.cs

```diff
@@ -51,8 +68,48 @@ private void RetryCreatePlayerRequest()
         protected override void OnUpdate()
         {
             if (PlayerLifecycleConfig.AutoRequestPlayerCreation && !initializationGroup.IsEmptyIgnoreFilter)
+            {
+                sendAutoPlayerCreationRequest = true;
+            }
+
+            if (playerCreatorEntityIds.Count == 0)
```

Comment:
Invert the if to reduce nesting. If this is `0` I don't think we should even attempt to send the request?

Reply:
atm we don't as there's a return at the end of this `if`

Reply:
Ah right - still would be cleaner if inverted